assets/section-featured-blog.css

```diff
@@ -1,5 +1,5 @@
 .blog:not(.background-secondary) {
-  margin: 5rem 0;
+  margin: calc(5rem + var(--page-width-margin)) 0;
 }
```

Comment:
In this file, and not related to a change in this PR it seems, there is `.blog:not(.no-heading)` which applies a negative margin-top on mobile but when you have to secondary background sections after another it creates this weird thing: [screenshot](https://screenshot.click/21-09-ka3kv-3fjps.png). Not sure what it was meant for 🤔

Reply:
I don't think so cause it's using a background that isn't used elsewhere 🤔

Reply:
@ludoboludo Updated to be desktop only 👍